_posts/2023-03-29-h2oai.md

```diff
@@ -0,0 +1,82 @@
+| groupby #10 |   `SELECT id1, id2, id3, id4, id5, id6, sum(v3) AS v3, count(*) AS count FROM tbl GROUP BY id1, id2, id3, id4, id5, id`  | Many many small groups                                                   |
+| join #1 |`SELECT x.*, small.id4 AS small_id4, v2 FROM x JOIN small USING (id1)`                                                                           |   join on a small table                                                                                                                                  |
+| join #2 |`SELECT x.*, medium.id1 AS medium_id1, medium.id4 AS medium_id4, medium.id5 AS medium_id5, v2 FROM x JOIN medium USING (id2)`                    |   inner join on a medium table                                                                                                           |
+| join #3 |`SELECT x.*, medium.id1 AS medium_id1, medium.id4 AS medium_id4, medium.id5 AS medium_id5, v2 FROM x LEFT JOIN medium USING (id2)`               |   left join on a medium table                                                                                                          |
+| join #4 |`SELECT x.*, medium.id1 AS medium_id1, medium.id2 AS medium_id2, medium.id4 AS medium_id4, v2 FROM x JOIN medium USING (id5)`                    |   join on a medium table using int                                                                                                      |
+| join #5 |`SELECT x.*, big.id1 AS big_id1, big.id2 AS big_id2, big.id4 AS big_id4, big.id5 AS big_id5, big.id6 AS big_id6, v2 FROM x JOIN big USING (id3)` |   join on a big table                                                                                                |
+
+
+### Modifications to the Benchmark & Hardware
+No modifications have been made to the queries or the data generation. Some scripts required minor modifications so that the current version of the library could be run. The hardware used is slightly different as the exact AWS offering the benchmark previously used is no longer available. Base libraries have been updated as well. GPU libraries were not tested. 
+
+
+AWS is a [m4.10xlarge](https://aws.amazon.com/ec2/instance-types/)
+
+- CPU model: Intel(R) Xeon(R) CPU E5-2676 v3 @ 2.40GHz 
+- CPU cores: 40 
+- RAM model: Unknown 
+- Memory: 160GB 
+- NO GPU specifications 
+- R upgraded, 4.0.0 -> 4.2.2 
+- Python upgraded 3.\[6\|7\] -> 3.10 
+
+### Changes made to install scripts of other systems
+Pandas, Polars, Dask, and Clickhouse required changes to their setup/install scripts. The changes were relatively minor consisting mostly of syntax updates and data ingestion updates. Data ingestion did not affect the reporting timing results.
+
+
+## RESULTS
+
+
+<iframe src="https://tmonster.github.io/h2oai-db-benchmark/"  title="h2o db benchmmark" height=500px width=800px></iframe>
```

Comment:
I can't get this to work because it's looking for an html file inside `_include`. The example you sent me on discord also uses actual html syntax an not the templated `{% %}` you suggested. I think keeping it the way it is is fine. I've been able to verify the format running the website locally using Jekyll